target_chains/solana/examples/send_usd/app/src/App.tsx

```diff
@@ -0,0 +1,129 @@
+import logo from "./logo.svg";
+import "./App.css";
+import {
+  AnchorWallet,
+  ConnectionProvider,
+  WalletProvider,
+  useAnchorWallet,
+} from "@solana/wallet-adapter-react";
+import {
+  WalletDisconnectButton,
+  WalletModalProvider,
+  WalletMultiButton,
+} from "@solana/wallet-adapter-react-ui";
+import {
+  PythSolanaReceiver,
+  InstructionWithEphemeralSigners,
+} from "@pythnetwork/pyth-solana-receiver";
+import { Connection, PublicKey } from "@solana/web3.js";
+import * as buffer from "buffer";
+import { AnchorProvider, BN, Program, Wallet } from "@coral-xyz/anchor";
+import { SendUSDApp, IDL } from "./idl/send_usd_app";
+import { PriceServiceConnection } from "@pythnetwork/price-service-client";
+window.Buffer = buffer.Buffer;
+
+require("@solana/wallet-adapter-react-ui/styles.css");
+
+const SOL_PRICE_FEED_ID =
+  "0xef0d8b6fda2ceba41da15d4095d1da392a0d2f8ed0c6c7bc0f4cfac8c280b56d";
+const ETH_PRICE_FEED_ID =
+  "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace";
+const BTC_PRICE_FEED_ID =
+  "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43";
+const BOBA_PRICE_FEED_ID =
+  "0xd1e9cff9b8399f9867819a3bf1aa8c2598234eecfd36ddc3a7bc7848432184b5";
+async function postPriceUpdate(connection: Connection, wallet?: AnchorWallet) {
+  if (wallet === undefined) {
+    return;
+  } else {
+    const priceServiceConnection = new PriceServiceConnection(
+      "https://hermes.pyth.network/",
+      { priceFeedRequestConfig: { binary: true } }
+    );
+    const pythSolanaReceiver = new PythSolanaReceiver({
+      connection,
+      wallet: wallet as Wallet,
+    });
+
+    const priceUpdateData = await priceServiceConnection.getVaa(
```

Comment:
Can you replace this by 
`const priceUpdateData = await priceServiceConnection.getLatestVaas([SOL_PRICE_FEED_ID])`

Reply:
Done.